README.md

```diff
@@ -1,50 +1,84 @@
-# Feel Tracker App
+<h1 align="center">Sentispace</h1>
+<p align="center">
+  <a href="https://sentispace.com/">
+    <img alt="Sentispace" title="Sentispace" src="https://i.imgur.com/xoTQj8f.png" width="150">
+  </a>
+</p>
 
-A diary app to track your daily activities, and how you feel about them.
-Feel Tracker will provide you with a day, week, and month view of the activities you want to track, as well as trending and statistics.
+## Table of Contents
+
+-   [About](#about)
+-   [Origin Story](#origin-story)
+-   [Features](#features)
+-   [How to contribute](#how-to-contribute)
+-   [Acknowledgements](#acknowledgements)
+-   [License](#license)
+
+## About
+
+Sentispace is an installable PWA diary app that's built with privacy in mind. You can log your daily activities and feelings, and the app provides summaries and trends for the day, week, or month. One of its key features is caching for offline access, allowing you to interact with your diary even when you're not connected to the internet. Everything runs on the user's own device—no back-end involved. The app is built using Vite, React, TypeScript and Dexie.
+
+-   [ORS/FRS Definitions](https://docs.google.com/document/d/11IM741stFVj3_9otDXg5aKKYirAFxMybJnWy4EoPHIQ/view?usp=sharing)
+-   [Architecture Diagram](https://drive.google.com/file/d/1uRsWsnlG9A36mTwNWcML81U5gVK3qsg2/view?usp=sharing)
+-   [UX Design on Figma](https://www.figma.com/file/JC406azeeeGGlTbLAxCs04/Feeling-Tracker?node-id=0%3A1&t=jyrThFgzn0f7LWof-0)
 
 ## Origin Story
 
-Due to the worsening of my chronic medical conditions, I had to start tracking my daily activity. 
+My chronic health conditions got worse, so I needed to track my daily activities closely. I couldn't find any existing apps that met my specific privacy and functionality needs, so I created my own. After sharing with friends, I saw a greater need and chose to release it as an open-source app for everyone.
+
+## Features
+
+-   Activity Management: Create, modify, and delete activities with ease.
+-   Category Management: Define, edit, and organize categories for activities.
+-   Organize with Categories: Assign activities to specific categories for enhanced tracking.
```

Comment:
Category management is kinda redundant if you already mention Organize with Categories. It's a given that you should be able to manage them.